src/firefly/js/visualize/PlotState.js

```diff
@@ -141,26 +141,22 @@ export class PlotState {
      *
      * @return {RotateType}
      */
-    getRotateType() {return this.rotationType; }
+    // getRotateType() {return this.rotationType; }
 
     /**
      * @summary check to see it the image is rotated
      * @return {boolean}
      * @public
      */
-    isRotated() {return this.rotationType!==RotateType.UNROTATE;}
+    // isRotated() {return this.rotationType!==RotateType.UNROTATE;}
 
     /**
      *
      * @return {boolean}
      */
     isFlippedY() { return this.flippedY; }
 
-    /**
-     *
-     * @return {Number}
-     */
-    getRotationAngle() { return this.rotationAngle; }
+    // getRotationAngle() { return this.rotationAngle; }
```

Comment:
How should can we get the rotation angle now? (I know we rely on PlotState a lot in API.)